core-web/apps/dotcms-ui/src/app/view/components/_common/dot-page-selector/dot-page-selector.component.ts

```diff
@@ -49,7 +49,6 @@ enum SearchType {
 })
 export class DotPageSelectorComponent implements ControlValueAccessor {
     @Output() selected = new EventEmitter<DotPageAsset | string>();
-    @Input() label: string;
```

Comment:
Don't you need to update the tests for this?

Reply:
I already updated this test case.